### PR TITLE
[PHP CS Fixer] Replace route use

### PR DESCRIPTION
**Pull request by JeremieSamson**
This new rule `App/sensio_to_symfony_route` will remove all `use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;` to `use Symfony\Component\Routing\Annotation\Route; `

### CODE REVIEW

File: src/Controller/Admin/AdminAdherentController.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 namespace AppBundle\Controller\Admin;
 
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\Routing\Annotation\Route;
```
[the remark this answers is not in the document]
Comment (ottaviano) [reply src/Controller/Admin/AdminAdherentController.php]: I admit it's not really easy to do, we can use this with `OrderedImportsFixer` of PHPCSFixer.

[the remark this answers is not in the document]
Comment (JeremieSamson) [reply src/Controller/Admin/AdminAdherentController.php]: Yep good idea !

File: src/Fixer/SensioToSymfonyRouteFixer.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        $namespace = implode('',
+            array_map(
+                function ($token) {
+                    return $token->getContent();
+                },
+                $tokens
+            )
+        );
+
+        return 'use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;' === $namespace;
```
[the remark this answers is not in the document]
Comment (JeremieSamson) [reply src/Fixer/SensioToSymfonyRouteFixer.php]: sure :)

File: .php_cs.dist [hunk 3]
```diff
@@ -34,6 +35,7 @@ return PhpCsFixer\Config::create()
         new PedroTroller\CS\Fixer\CodingStyle\LineBreakBetweenMethodArgumentsFixer,
         new PedroTroller\CS\Fixer\CodingStyle\LineBreakBetweenStatementsFixer,
         new PedroTroller\CS\Fixer\Comment\UselessCommentFixer,
-        new AppBundle\Fixer\DoctrineMigrationCleanFixer
+        new AppBundle\Fixer\DoctrineMigrationCleanFixer,
+        new AppBundle\Fixer\SensioToSymfonyRouteFixer
```
Comment (ottaviano) [inline .php_cs.dist]: `Fixer,` 😄

**Pull request merged**